src/box/lua/config/configdata.lua

```diff
@@ -210,7 +210,7 @@ function methods.sharding(self)
     local cfg = {
         sharding = sharding,
         box_cfg_mode = 'manual',
-        schema_management_mode = 'auto',
+        schema_management_mode = 'manual_access',
```

Comment:
Nit: I would appreciate a few words about a reson behind using this option here (in the code).

Reply:
Added a comment.

Reply:
I meant a bit different thing. Here you said that this mode allows to manage everything manually. However, the most interesting (and non-obvious) fact is why we need it. AFAIU, there are two reasons:

* To sync privilige management with applier/credentials.lua.
* To deploy the functions on RO->RW transition -- it is important if all instances of a replicaset are started in RO (say, in case of `replication.failover: election`).

Feel free to ignore. I just want some record about reasons of our decisions.

Reply:
Thanks for the suggestion! I fixed the comment and rebased on the master.

Reply:
Thank you for the update!